source/fundamentals/transactions.txt

```diff
@@ -208,6 +208,15 @@ the Convenient Transaction API:
 To see a fully runnable example that uses this API, see the
 :ref:`node-usage-convenient-txn` usage example.
 
+.. sharedinclude:: dbx/transactions-parallelism.rst
```

Comment:
Why is this joined with:
> Parallel Operations Not Supported
> The Node.js driver does not support running parallel operations within a single transaction. 

It seems like a separate note to point folks at the client api for more than one namespace. Also, none of the driver support parallel operations in a transaction because it isn't supported on the server rather than there being a driver side limitation.

Reply:
Let me check with the docs team on that, we pretty much got tickets for most of the drivers docs that have a transactions page to add this and organize it this way. But I'll link your comment and see what the feedback is!